packages/internal-test-env/rollup.config.mjs

```diff
@@ -0,0 +1,69 @@
+// The following is only possible from Node 18 onwards
+// import pkg from "./package.json" assert { type: "json" };
+
+// Until we only support Node 18+, this should be used instead
+// (see https://rollupjs.org/guide/en/#importing-packagejson)
+import { createRequire } from "node:module";
+const require = createRequire(import.meta.url);
+const pkg = require("./package.json");
+
+import typescript from "rollup-plugin-typescript2";
+
+const external = [
+  ...Object.keys(pkg.dependencies || {}),
+  "@inrupt/solid-client-authn-node",
+  "@inrupt/solid-client",
+];
+
+const plugins = [
+  typescript({
+    // Use our own version of TypeScript, rather than the one bundled with the plugin:
+    typescript: require("typescript"),
+    tsconfigOverride: {
+      compilerOptions: {
+        module: "esnext",
+      },
+    },
+  }),
+];
+
+const rollupDefaultConfig = { external, plugins };
+
+export default [
+  {
+    input: "index.ts",
+    output: [
+      {
+        file: pkg.main,
+        format: "cjs",
+      },
+      {
+
+        file: pkg.module,
+        entryFileNames: "[name].es.js",
+        format: "esm",
+      },
+    ],
+    ...rollupDefaultConfig,
+  },
+  {
+    input: ["index.ts"],
+    output: {
+      dir: "dist",
+      entryFileNames: "[name].mjs",
+      format: "esm",
+      preserveModules: true,
+    },
+    ...rollupDefaultConfig,
```

Comment:
Would it make sense to have the default spread at the top of the object, so that it is overridden by any subsequent definition, rather that it overriding things explicitly defined?